src/create_wand_data.cpp

```diff
@@ -41,6 +43,7 @@ int main(int argc, const char **argv)
     app.add_flag("--compress", compress, "Compress additional data");
     app.add_option("-s,--scorer", scorer_name, "Scorer function")->required();
     app.add_flag("--range", range, "Create docid-range based data")->excludes(var_block_opt);
```

Comment:
Maybe we should define a criteria regarding the use of the `document id` and the `term id `concepts, specially in the usage info. For example, `docid/doc ID/document identifier/etc` and `termid/term ID/term identifier/etc` are used respectively.